### DISCUSSION TITLE
hallo mau tanya

### DISCUSSION THREAD

**Discussion by ralfy26** [dewanakl/undangan-api#18] [Q&A] [answered]
gimana cara runningnya ya mas dewa? kok udah ngikutin caranya muncul kayak gini

 PS C:\xampp\htdocs\undangan-api-main> cp .env.example .env
PS C:\xampp\htdocs\undangan-api-main> composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Package operations: 21 installs, 0 updates, 0 removals
    Failed to download symfony/deprecation-contracts from dist: The zip extension and unzip/7z commands are both missing, skipping.
The php.ini used by your command-line PHP is: C:\xampp\php\php.ini
    Now trying to download from source

In GitDownloader.php line 82:

  git was not found in your PATH, skipping source download  
                                                            

install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--download-only] [--dev] [--no-suggest] [--no-dev] [--no-autoloader] [--no-progress] [--no-install] [--audit] [--audit-format AUDIT-FORMAT] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>...]

PS C:\xampp\htdocs\undangan-api-main> php saya key
PHP Warning:  require_once(C:\xampp\htdocs\undangan-api-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\undangan-api-main\saya on line 11

Warning: require_once(C:\xampp\htdocs\undangan-api-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\undangan-api-main\saya on line 11
PHP Fatal error:  Uncaught Error: Failed opening required 'C:\xampp\htdocs\undangan-api-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\undangan-api-main\saya:11
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\undangan-api-main\saya on line 11

Fatal error: Uncaught Error: Failed opening required 'C:\xampp\htdocs\undangan-api-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\undangan-api-main\saya:11
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\undangan-api-main\saya on line 11
PS C:\xampp\htdocs\undangan-api-main> php saya migrasi --gen
PHP Warning:  require_once(C:\xampp\htdocs\undangan-api-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\undangan-api-main\saya on line 11

Warning: require_once(C:\xampp\htdocs\undangan-api-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\undangan-api-main\saya on line 11
PHP Fatal error:  Uncaught Error: Failed opening required 'C:\xampp\htdocs\undangan-api-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\undangan-api-main\saya:11
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\undangan-api-main\saya on line 11

Fatal error: Uncaught Error: Failed opening required 'C:\xampp\htdocs\undangan-api-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\undangan-api-main\saya:11
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\undangan-api-main\saya on line 11
PS C:\xampp\htdocs\undangan-api-main> php saya coba
PHP Warning:  require_once(C:\xampp\htdocs\undangan-api-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\undangan-api-main\saya on line 11

Warning: require_once(C:\xampp\htdocs\undangan-api-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\undangan-api-main\saya on line 11
PHP Fatal error:  Uncaught Error: Failed opening required 'C:\xampp\htdocs\undangan-api-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\undangan-api-main\saya:11
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\undangan-api-main\saya on line 11

Fatal error: Uncaught Error: Failed opening required 'C:\xampp\htdocs\undangan-api-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\undangan-api-main\saya:11
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\undangan-api-main\saya on line 11

**Comment by dewanakl** [comment, marked as answer]
Untuk ekstension zip pada php di enable dulu mas

**Reply by dewanakl** [reply]
Kemudian coba composer install lagi sampai semuanya sudah ke install

**Reply by ralfy26** [reply]
baik terimakasih ya mass

**Reply by dewanakl** [reply]
Pastikan menggunakan php 8.1 keatas

**Reply by dewanakl** [reply]
Sama sama mas
